### PR TITLE
Mobilified homepage + other fixes

### PR DESCRIPTION
**Pull request by andylouisqin**
Should work now on phones and tablets. Intended to fix issues: #281 #276 #273 #272 #269

### CODE REVIEW

File: revolv/templates/base/home.html [hunk 1]
```diff
@@ -60,7 +60,14 @@
...
             </a>
         </div>
-        <video autoplay loop muted src="{% static "videos/cover.mp4" %}" class="cover-video"></video>
+        <!-- Background video to play on desktop view -->
+        <video autoplay loop muted poster="{% static "images/cover-video-poster.png" %}" class="cover-video">
+            <source src="{% static "videos/cover.mp4" %}" type="video/mp4">
+            <source src="{% static "videos/cover.webm" %}" type="video/webm">
+        </video>
+        <!-- Background image displayed on mobile devices -->
+        <img src="{% static "images/mobile-cover.png" %}"
```
Comment (noahsark769) [inline revolv/templates/base/home.html]: This is a super nitpick, but we don't really need this img tag to be on separate lines when it's shorter than some elements which are on one line above

**Comment by noahsark769** [comment]
Wow, this looks awesome! So no way to make the video play on tablet/mobile I guess? Two quick things:

1. The text displayed on the homepage project feature is the org's mission right now, and I think it should be the project description instead, right?
2. On mobile (landscape and portrait) I think that the title of the project in the feature should be centered instead of left aligned, to match how it is on the project page.

Other than that this looks great to me, almost ready to ship!

**Comment by andylouisqin** [comment]
Thanks for the feedback @noahsark769! Just made all the changes.

**Comment by noahsark769** [comment]
Sweet, this looks great! So many issues closed in one pull request!

```
___________________________________________________________________
||     *                            *                    ((   *  ||
||        *                 *                *            ~      ||
||                ___.                          *          *     ||
||       *    ___.\__|.__.           *                           ||
||            \__|. .| \_|.                                      ||
||            . X|___|___| .                         *           ||
||          .__/_||____ ||__.            *                /\     ||
||  *     .  |/|____ |_\|_ |/ _                          /  \    ||
||        \ _/ |_X__\|_  |\||~,~{                       /    \   ||
||         \/\ |/|    |_ |/:|`X'{                   _ _/      \__||
||          \ \/ |___ |_\|_.|~~~                   /    . .. . ..||
||         _|X/\ |___\|_ :| |_.                  - .......... . .||
||         | __\_:____ |  ||o-|            ___/........ . . .. ..||
||         |/_-|-_|__ \|_ |/--|       ____/  . . .. . . .. ... . ||
|| ........:| -|- o-o\_:_\|o-/:....../...........................||
|| ._._._._._\=\====o==o==o=/:.._._._._._._._._._._._._._._._._._||
|| _._._._._._\_\ ._._._._.:._._._._._._._._._._._._._.P_._._._._||
|| ._._._._._._._._._._._._._._._._._._._._._._._._._._._._._._._||
||---------------------------------------------------------------||
```

**Pull request merged**